automl/feature-selection/script.whizzml

```diff
@@ -52,22 +60,25 @@
   (when (= "dataset" (resource-type train-dataset))
     (create-dataset {"name" (str (resource-name train-dataset) " | filtered")
                      "origin_dataset" train-dataset
-                     "input_fields" selected-fields})))
+                     "input_fields" (field-ids-from-names train-dataset
```

Comment:
What if the field name is not found in the dataset fields? Isn't that going to cause trouble?